costmodel/router.go

```diff
@@ -533,35 +532,40 @@ func (a *Accesses) AggregateCostModel(w http.ResponseWriter, r *http.Request, ps
 	} else {
 		start := startTime.Format(RFC3339Milli)
 		end := endTime.Format(RFC3339Milli)
-		costData, err = a.Model.ComputeCostDataRange(pClient, a.KubeClientSet, a.Cloud, start, end, resolution, "", "", remoteEnabled)
+		costData, err = A.Model.ComputeCostDataRange(pClient, A.KubeClientSet, A.Cloud, start, end, resolution, "", "", remoteEnabled)
 		if err != nil {
 			w.Write(WrapData(nil, err))
 			return
 		}
 
-		a.CostDataCache.Set(key, costData, cache.DefaultExpiration)
+		A.CostDataCache.Set(key, costData, cache.DefaultExpiration)
 	}
 
-	c, err := a.Cloud.GetConfig()
+	c, err := A.Cloud.GetConfig()
 	if err != nil {
 		w.Write(WrapData(nil, err))
 		return
 	}
-	discount, err := parsePercentString(c.Discount)
+	discount, err := strconv.ParseFloat(c.Discount[:len(c.Discount)-1], 64)
```

Comment:
niko just added this. why are we removing?

Reply:
Fixed, uses ParsePercentString now.